File: app/models/wkbk/answer_kind_info.rb
module Wkbk
  class AnswerKindInfo
    include ApplicationMemoryRecord
    memory_record [
      { key: :correct, name: "正解",   mark: "O", },
      { key: :mistake, name: "不正解", mark: "X", },
    ]
  end
end

File: nuxt_side/components/Wkbk/models/answer_kind_info.js
import { ApplicationMemoryRecord } from "@/components/models/application_memory_record.js"

export class AnswerKindInfo extends ApplicationMemoryRecord {
  static get define() {
    return [
      { key: "correct", name: "正解",   sound_key: "o", name: "正解", icon: "circle-outline", type: "is-danger",  },
      { key: "mistake", name: "不正解", sound_key: "x", name: "誤答", icon: "close",          type: "is-success", },
    ]
  }

  get icon_attrs() {
    return { icon: this.icon, type: this.type }
  }
}
